remote/github/github.go

```diff
@@ -249,6 +256,21 @@ func (g *Github) Status(u *model.User, r *model.Repo, b *model.Build, link strin
 	return err
 }
 
+func deploymentStatus(client *github.Client, r *model.Repo, b *model.Build, link string) error {
+	// the deployment ID is only available in the the link to the build as the last element in the URL
+	parts := strings.Split(b.Link,"/")
+	id, _ := strconv.Atoi(parts[len(parts)-1])
+	status := getStatus(b.Status)
```

Comment:
yes, when the deployment link does not match we should exit without failure. This will indicate a deployment was triggered by something other than GitHub in which case no deployment status should be sent to GitHub.

```diff
+var re = regexp.MustCompile(".+/deployment/(\\d)")

func deploymentStatus(client *github.Client, r *model.Repo, b *model.Build, link string) error {
+	matches := re.FindStringSubmatch(b.Link)
+	if len(matches) != 2 {
+		return nil
+	}
+	id, _ := strconv.Atoi(matches[1])

```